src/garage/tf/baselines/gaussian_cnn_baseline.py

```diff
@@ -7,10 +7,10 @@
 from garage import make_optimizer
 from garage.misc.tensor_utils import normalize_pixel_batch
 from garage.np.baselines.baseline import Baseline
+from garage.tf.baselines.gaussian_cnn_regressor_model import (
```

Comment:
can you rename this module/class `GaussianCNNRegressorModel`?

Reply:
Renamed